technologies/app/saagie-code-search/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+version: v1
+type: APP
+id: saagie-code-search
+label: Saagie Code Search
+baseline: "Saagie Code Search"
+description: "Saagie-Code-Search: Custom app based on Flask that Saagie Code Search is a web application that enables the search for Python functions, either within a default codebase or GitHub repositories."
+available: true
+icon: datascience
+backgroundColor: "#253c5b"
+customFlags: []
+contexts:
+  - id: saagie-code-search-0.1
+    label: For Saagie V1
+    releaseNotes: ""
+    available: true
+    trustLevel: experimental
+    ports:
+      - port: 5000
+        name: myapp
+        rewriteUrl: false
+        basePath: SAAGIE_BASE_PATH
+    volumes: [ "/data" ]
+    dockerInfo:
+      image: saagie/saagie-code-search
+      version: 0.1
```

Comment:
please put the value of version into double quote, because it can be interpret like a float:

`version: "0.1"`

Reply:
Done